src/libraries/System.Private.CoreLib/src/System/IO/FileSystem.Windows.cs

```diff
@@ -602,7 +618,7 @@ uint GetFinalPathNameByHandle(SafeFileHandle handle, char[] buffer)
             {
                 fixed (char* bufPtr = buffer)
                 {
-                    return Interop.Kernel32.GetFinalPathNameByHandle(handle, bufPtr, (uint)buffer.Length, Interop.Kernel32.FILE_NAME_NORMALIZED);
+                    return Interop.Kernel32.GetFinalPathNameByHandle(handle, bufPtr, (uint)buffer.Length, Interop.Kernel32.FILE_NAME_OPENED);
```

Comment:
I'm reverting this. I described the difference between `FILE_NAME_NORMALIZED` and `FILE_NAME_OPENED` [here](https://github.com/dotnet/runtime/issues/57582#issuecomment-900624728), but I discovered the behavior seemed to be the same: both were returning fully normalized paths.

We asked the Windows team that owns this API, and it seems the difference is that for this P/Invoke, passing `FILE_NAME_NORMALIZED` would return the full normalized name, while `FILE_NAME_OPENED` would return a shorted version of that same normalized name, if it happened to be too long.